stubs/Php81.phpstub

```diff
@@ -6,6 +6,16 @@ namespace {
     }
 }
 
+namespace {
+    class CURLStringFile {
+        public string $data;
+        public string $postname;
+        public string $mime;
+
+        public function __construct(string $data, string $postname, string $mime = "application/octet-stream") {}
```

Comment:
unless you have specific annotations to add, this should not be needed. The __construct signature is redundant with the CallMap and the properties signatures should be detailed in PropertyMap.php